vms/job/templates/job/list.html

```diff
@@ -42,5 +211,6 @@
         <div class="alert alert-success">
             {% trans "There are currently no jobs. Please create jobs first." %}
```

Comment:
This is not an appropriate message. Instead it should be like `There are currently no jobs matching your search query`